src/main/java/de/bonndan/nivio/input/dto/ItemDescription.java

```diff
@@ -223,6 +225,11 @@ public void setLifecycle(Lifecycle lifecycle) {
         this.lifecycle = lifecycle;
     }
 
+    public void setLifecycle(String lifecycle) {
+        if (!StringUtils.isEmpty(lifecycle))
+            setLifecycle(Lifecycle.from(lifecycle));
```

Comment:
Müsste hier nicht noch etwas im Fehlerfall passieren? Ich finde es etwas merkwürdig, wenn ich eine Methode mit ungültigem Parameter aufrufe und diese still und heimlich so tut als wäre alles in Ordnung :see_no_evil:

Reply:
The idea is to have nivio as graceful and forgiving as possible in order to keep frustration low. That being said, behaviour might be improved here. What do you suggest?